oauth2_http/java/com/google/auth/oauth2/ExternalAccountSupplierContext.java

```diff
@@ -0,0 +1,78 @@
+package com.google.auth.oauth2;
+
+import com.google.errorprone.annotations.CanIgnoreReturnValue;
+import java.io.Serializable;
+
+/** Context object to pass relevant variables from external account credentials to suppliers. */
```

Comment:
nit: Let's explain how this is used with an example.

Reply:
Added details on how it is used, not sure what you are looking for as an example? Just an example of what one of the supplier function calls looks like?